drf_stripe/stripe_api/checkout.py

```diff
@@ -114,6 +114,15 @@ def _make_stripe_checkout_params(
             "trial_end": int(_make_trial_end_datetime(trial_end=trial_end).timestamp())
         }
     }
+    
+    allow_promotion_codes = drf_stripe_settings.ALLOW_PROMOTION_CODES
+    
+    if allow_promotion_codes:
+        ret.update({"allow_promotion_codes": allow_promotion_codes})
+    else:
+        ret.update({"discounts": discounts if discounts else drf_stripe_settings.ALLOW_PROMOTION_CODES})
```

Comment:
is it supposed to be `drf_stripe_settings.DEFAULT_DISCOUNTS` here that's passed into `"discount"` key?